pages.pl/common/tldrl.md

```diff
@@ -0,0 +1,8 @@
+# tldrl
+
+> This command is an alias of `tldr-lint`.
```

Comment:
Did you forget this page?
```suggestion
> Te polecenie jest aliasem `tldr-lint`.
```

Reply:
Yes, it looks like I translated this but broke the commit and threw in the untranslated one.  
Fixing this now, sorry.

Btw, it should be "To polecenie", not "Te polecenie".